src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -440,18 +440,22 @@ export default class WebGLRenderer extends SystemRenderer
      * Changes the current shader to the one given in parameter
      *
      * @param {PIXI.Shader} shader - the new shader
+     * @param {boolean} [autoProject=true] - Whether automatically set the projection matrix
      * @return {PIXI.WebGLRenderer} Returns itself.
      */
-    bindShader(shader)
+    bindShader(shader, autoProject)
```

Comment:
No problem. Leave an inline comment:

```js
// `autoProject` normally would be a default parameter set to true
// but because of how Babel transpiles default parameters
// it hinders the performance of this method.
```